packages/ng-excel-builder/src/excel-builder.service.ts

```diff
@@ -91,20 +92,20 @@ export class ExcelBuilderService {
 
                 // eslint-disable-next-line max-lines-per-function
                 function generateRow(
-                    worksheets: Any,
-                    data: Any,
-                    titles: Any,
-                    rowHeight: Any,
-                    ptWidth: Any,
-                    colWidth: Any,
-                    styleType: Any
+                    worksheet: ExcelWorksheet<T>,
+                    entries: T[],
+                    titles: string[],
+                    rowHeight: number,
+                    ptWidth: number,
+                    colWidth: number,
+                    styleType: PlainObject
                 ) {
```

Comment:
is this file ignored by linter? where is the return type here and below?

Reply:
yes, it will be fixed later

Reply:
👀